tests/Doctrine/Tests/Common/Cache/PhpFileCacheTest.php

```diff
@@ -66,6 +66,17 @@ protected function _getCacheDriver()
     {
         return new PhpFileCache($this->directory);
     }
+
+    public function testFailureOnIncludeNotSettedCacheDoesntGetCaughtAsError()
+    {
+        $cache = $this->_getCacheDriver();
+        set_error_handler(function($errno, $errstr, $errfile, $errlin){
+            $this->fail('include failure captured');
```

Comment:
If that will be triggered restore_error_handler won't be called, causing many other tests to fail as well.